moped-editor/src/routes.js

```diff
@@ -37,6 +38,7 @@ const routes = [
       { path: "staff/edit/:userId", element: <EditStaffView /> },
       { path: "projects", element: <ProjectsListView /> },
       { path: "projects/new", element: <NewProjectView /> },
+      { path: "projects/new/success", element: <ProjectSummary /> },
```

Comment:
I like the success route, but it would be more efficient if we changed it tom something like `projects/:projectId`, that way we can use that same page for any project and not just the one that has been created. This may be a good question for @mateoclarke and @mddilley

Reply:
sounds like a plan.

Reply:
Yeah, I agree the `projects/:projectId` should be the pattern instead of a dedicated success page. Going to work on this update tonight.